src/main/java/seedu/address/model/task/Priority.java

```diff
@@ -8,8 +8,8 @@
  */
 public class Priority {
 
-    public static final String MESSAGE_PRIORITY_CONSTRAINTS = "Task priority should only contain numbers";
-    public static final String PRIORITY_VALIDATION_REGEX = "\\d+";
+    public static final String MESSAGE_PRIORITY_CONSTRAINTS = "Task priority can take any values";
```

Comment:
This one is tricky, but should we allow users to input any value or a numeric value from 1 to 5?